### ISSUE TITLE
Post a blog on every release

### ISSUE THREAD

**Issue by surajssd** [kedgeproject/kedge#339]
We(mostly @cdrage) used to do this on every release of atomicapp.

For every release we should post blog on some platform. 

To make the work less, maintain a google doc for that where team contributes and blog post goes out on the day of release with release.

**Comment by cdrage** [comment]
@surajssd 

I agree. Our main site would be *great* for it (http://kedgeproject.org).
